### PR TITLE
fix: dont npe when no kubeconfig/current context (#630)

### LINKED ISSUE #630: Can't login when no kube context exists
When no .kube/config file exists, or is empty, the extension shows an error:

Error: Cannot invoke "io.fabric8.kubernetes.api.model.NamedContext.getName()" because return value of "com.redhat.devtools.intellij.common.utils.ConfigHelper.getCurrentContext()" is null 
(I think some user friendly message would be better)

While this error exists, I cannot login to any cluster as the login window won't even appear.

### PR DESCRIPTION
**Pull request by adietish**
fixes #630 
depends on https://github.com/redhat-developer/intellij-common/pull/204

### CODE REVIEW

**Comment by adietish** [comment]
There are 2 parts in this fix:
1. dont NPE when trying to retrieve the current context
2. display a user friendly node text if there's no current context/no kube config
image

@sbouchet: please review

**Comment by adietish** [comment]
https://github.com/redhat-developer/intellij-common/pull/204 was merged by resulting library no published to the maven repo yet, apparently (compilation fails).

File: src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java [hunk 1]
```diff
@@ -149,9 +150,8 @@ public OdoCli(com.intellij.openapi.project.Project project, String command) {
         this.command = command;
         this.project = project;
         this.connection = ApplicationManager.getApplication().getMessageBus().connect();
-        String context = ConfigHelper.getCurrentContext().getName();
-        Config config = Config.autoConfigure(context);
-
+        String current = ConfigHelper.getCurrentContextName();
```
[the remark this answers is not in the document]
Comment (adietish) [reply src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java:153]: this was added in https://github.com/redhat-developer/intellij-common/pull/204 which is 1.9.3-SNAPSHOT. It's apparently not published to repository.jboss.org yet :(

**Review by sbouchet — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
@sbouchet: released intellij-common 1.9.3 (Final), bumped this PR to use it.

**Pull request merged**